modules/partition-replicator/src/main/java/org/apache/ignite/internal/partition/replicator/raft/ZonePartitionRaftListener.java

```diff
@@ -68,18 +72,27 @@ public class ZonePartitionRaftListener implements RaftGroupListener {
     /** Mapping table ID to table request processor. */
     private final Map<Integer, RaftTableProcessor> tableProcessors = new ConcurrentHashMap<>();
 
+    private final TxStatePartitionStorage txStateStorage;
+
     private final PartitionsSnapshots partitionsSnapshots;
 
     private final PartitionKey partitionKey;
 
     /**
-     * Latest committed configuration of the zone-wide Raft group.
+     * Last applied index across all table processors.
```

Comment:
all table processors and txnStatePartitionStorage itself?

Reply:
yes

Reply:
Ok, so please update javadoc.